src/parser.ts

```diff
@@ -581,74 +603,93 @@ export class Parser {
     if (!possibleStatements.length) {
       // if no class declaration is found, try to find a
       // expression statement used in a React.StatelessComponent
-      possibleStatements = source.statements.filter(stmt =>
-        ts.isExpressionStatement(stmt)
+      possibleStatements = source.statements.filter(
+        stmt => ts.isExpressionStatement(stmt) || ts.isVariableStatement(stmt)
       );
     }
 
-    if (!possibleStatements.length) {
-      return {};
-    }
-
-    const statement = possibleStatements[0];
+    return possibleStatements.reduce((res, statement) => {
```

Comment:
merging multiple definitions in case of defaultProps are used together with desctructuring